nifi-registry/nifi-registry-core/nifi-registry-docs/src/main/asciidoc/administration-guide.adoc

```diff
@@ -253,6 +253,63 @@ The `kerberos-identity-provider` has the following properties:
 
 See also <<kerberos_service>> to allow single sign-on access via client Kerberos tickets.
 
+[[openid_connect]]
+=== OpenID Connect
+
+OpenID Connect integration provides single sign-on using a specified Authorization Server.
+The implementation supports the Authorization Code Grant Type as described in
+link:https://www.rfc-editor.org/rfc/rfc6749#section-4.1[RFC 6749 Section 4.1^] and
+link:https://openid.net/specs/openid-connect-core-1_0.html#CodeFlowSteps[OpenID Connect Core Section 3.1.1^].
+
+After successful authentication with the Authorization Server, NiFi Registry generates an application Bearer Token with an
+expiration based on the OAuth2 Access Token expiration. NiFi Registry stores authorized tokens using the local State
+Provider and encrypts serialized information using the application Sensitive Properties Key.
+
```

Comment:
This paragraph does not apply in the same way to Registry, so it can be removed.
```suggestion
```